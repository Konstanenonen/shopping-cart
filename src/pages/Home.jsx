import React from 'react';
import styles from '../styles/pages/Home.module.scss';

function Home() {
  return (
    <>
      <h1 className={styles.title}>Home page</h1>
      <p className={styles.text}>
        This project was made to learn{' '}
        <span className={styles.red}>React testing library</span> and{' '}
        <span className={styles.red}>React Router library</span>
      </p>
      <p className={styles.text}>
        Checkout my other projects from my{' '}
        <a href="https://github.com/Konstanenonen">GitHub profile</a>
      </p>
    </>
  );
}

export default Home;
