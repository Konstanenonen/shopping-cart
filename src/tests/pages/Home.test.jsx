/* global it describe expect */

import React from 'react';
import { render } from '@testing-library/react';
import Home from '../../pages/Home';

describe('Home page', () => {
  it('renders information about learning goals and link to GitHub profile', () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
