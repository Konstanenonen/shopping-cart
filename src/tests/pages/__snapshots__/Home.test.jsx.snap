// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page renders information about learning goals and link to GitHub profile 1`] = `
<div>
  <h1
    class="title"
  >
    Home page
  </h1>
  <p
    class="text"
  >
    This project was made to learn
     
    <span
      class="red"
    >
      React testing library
    </span>
     and
     
    <span
      class="red"
    >
      React Router library
    </span>
  </p>
  <p
    class="text"
  >
    Checkout my other projects from my
     
    <a
      href="https://github.com/Konstanenonen"
    >
      GitHub profile
    </a>
  </p>
</div>
`;
